docs/usage/self-hosted-configuration.md

```diff
@@ -60,6 +55,9 @@ npm ci --ignore-scripts
 npx ng update @angular/core --from=10.0.0 --to=11.0.0 --migrate-only --allow-dirty --force
 ```
 
+If you wish to disable templating because of any security or performance concern, you may set `allowPostUpgradeCommandTemplating` to `false`.
+But before you disable templating completely, try the `allowedPostUpgradeCommands` config option, to limit what commmands are allowed to run.
```

Comment:
```suggestion
But before you disable templating completely, try the `allowedPostUpgradeCommands` config option to limit what commands are allowed to run.
```

Whoops, I had three `m`'s in the word `commands`. 😄 We probably don't need that `,` either.